### DISCUSSION TITLE
Troubleshooting why the log view in esphome for the DSCAlarm is polling the wrong IP

### DISCUSSION THREAD

**Discussion by fennemaj** [Dilbert66/esphome-dsckeybus#152] [Q&A]
I'm working to migrate from the older custom component model of this awesome solution, to the current version and I thought it would be easiest to just flash a new Wemos D1 mini and pop it into the board I have once it's set up, rather than flashing the existing one and potentially ending up with some challenges during the transition.

I flashed the new mcu and it looks good when looking at the log output when directly connected, however, when I try and look at the log from the ESPHome Device Builder screen it keeps looking for a log from 192.168.10.73, which is not the addressing that I use internally and I cannot for the life of me find where that IP is originating from.  I'm using the Dilbert github repo for the components, in case that's helpful to know.  Any thoughts or suggestions would be greatly appreciated.

Thanks so much!

**Comment by Dilbert66** [comment]
Sorry, I have no idea what that issue is. You have something in your api setup possibly that needs attention.  My code does not hard code any ip addresses. It uses whatever the yaml is setup with. Make sure each device has their own name.

**Reply by fennemaj** [reply]
Thanks for the quick response!  I really appreciate it.  I'll keep hunting around.  Thanks again and thanks for developing this!
